src/style.css

```diff
@@ -0,0 +1,90 @@
+@font-face {
+  src: url(fonts/Avenir-Book.ttf);
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.search-string {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  position: relative;
+  width: 100%;
+  margin: 20px 0;
+}
+
+.search-string:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-string:focus {
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.image-big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.input-line {
+  border: none;
+  outline: none;
+  width: 100%;
+  color: #3d4e61;
+}
+
+.big-input-line {
+  padding: 25px 0 23px 62px;
+  font-weight: 400;
+}
+
+.big-input-line::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  font-weight: 300;
+}
+
+.image-little {
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/98764641/197197821-e3f46d20-1363-48c0-a164-76b7863e6bec.png)

apply it for all code